common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1656,6 +1656,19 @@
     ],
     "sqlState" : "XX000"
   },
+  "INTERNAL_ERROR_TWS" : {
+    "message" : [
+      "Internal error using TransformWithStateOperator: "
+    ],
+    "subClass" : {
+      "IMPLICIT_KEY_NOT_FOUND" : {
```

Comment:
Internal errors should be not visible to users, so, don't need to introduce any sub-classes. Please, remove it and move `<message>` to `"Internal error using TransformWithStateOperator: "`